field_manager/field_manager.F90

```diff
@@ -560,6 +555,39 @@ subroutine field_manager_init(nfields, table_name)
 integer,                      intent(out), optional :: nfields    !< number of fields
 character(len=fm_string_len), intent(in),  optional :: table_name !< Name of the field table, default
 
+if (module_is_initialized) then
+   if(present(nfields)) nfields = num_fields
+   return
+endif
+
+call initialize
```

Comment:
This name is somewhat generic, should it be made more explicit?

Reply:
renamed to `initialize_module_variables`